README.md.txt

```diff
@@ -0,0 +1,84 @@
+# Collaborative Create-Your-Own-Adventure
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.